app/src/main/java/com/kms/cura/entity/LoginActivityEntity.java

```diff
@@ -0,0 +1,10 @@
+package com.kms.cura.entity;
+
+/**
+ * Created by duyhnguyen on 5/25/2016.
+ */
+public class LoginActivityEntity  extends Entity{
```

Comment:
No need to create LoginActivityEntity because it have login doesn't require data from Server